src/code/load.c

```diff
@@ -8,16 +8,17 @@ size_t Overlay_Load(uintptr_t vromStart, uintptr_t vromEnd, void* vramStart, voi
     s32 size = vromEnd - vromStart;
 
     if (gOverlayLogSeverity >= 3) {
-        // "Start loading dynamic link function"
-        PRINTF("\nダイナミックリンクファンクションのロードを開始します\n");
+        PRINTF(
+            T("\nダイナミックリンクファンクションのロードを開始します\n", "\nStart loading dynamic link function\n"));
```

Comment:
did the auto formatter do this :(

Reply:
CI didn't fail so yes this is in accordance with the formatter